build-system/release-tagger/index.js

```diff
@@ -21,21 +21,25 @@ const {publishRelease, rollbackRelease} = require('./update-release');
  * @return {Promise<void>}
  */
 async function _promote() {
-  const supportedChannels = ['beta-percent', 'stable', 'lts'];
+  const supportedChannels = ['beta-opt-in', 'beta-percent', 'stable', 'lts'];
   if (!supportedChannels.includes(channel)) {
     return;
   }
 
-  try {
+  if (channel == 'stable') {
     await publishRelease(head);
     log('Published release', head);
-  } catch (e) {
+  }
```

Comment:
nit: As a way to self-document the actual order of operations, maybe move this `Published release` block to after `Created release`?